easybuild/easyconfigs/d/daal/daal-2017.1.132.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "IntelBase"
+name = 'daal'
+version = '2017.1.132'
+
+homepage = 'http://software.intel.com/en-us/intel-daal'
+description = """Boost machine learning and big data analytics performance 
+ with easy-to-use library.
+ Features highly tuned functions for deep learning, classical 
+ machine learning, and data analytics performance across spectrum 
+ of Intel® architecture devices.
+ Optimizes data ingestion together with algorithmic computation for 
+ highest analytics throughput.
+ Includes Python*, C++, and Java* APIs and connectors to popular data 
+ sources including Spark* and Hadoop*.
+ Free and open source community-supported versions are available, as 
+ well as paid versions that include premium support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_%(name)s_%(version)s.tgz']
+
+dontcreateinstalldir = 'True'
+
+license_file = HOME + '/licenses/intel/license.lic'
+
+modextrapaths = { 
+	'LIBRARY_PATH':['compilers_and_libraries_2017.1.132/linux/daal/lib/intel64_lin','compilers_and_libraries_2017.1.132/linux/daal/../tbb/lib/intel64_lin/gcc4.4'],
+	'LD_LIBRARY_PATH':['compilers_and_libraries_2017.1.132/linux/daal/lib/intel64_lin','compilers_and_libraries_2017.1.132/linux/daal/../tbb/lib/intel64_lin/gcc4.4'],
+	'CPATH':'compilers_and_libraries_2017.1.132/linux/daal/include',
+	'CLASSPATH':'compilers_and_libraries_2017.1.132/linux/daal/lib/daal.jar'
```

Comment:
@shahzebsiddiqui please don't hardcode the version here, use `%(version)s` instead?